### PR TITLE
Use sls_sdk internally

### PR DESCRIPTION
**Pull request by selcukcihan**
### Description
Related issue https://linear.app/serverless/issue/SC-708/urgent-python-sdk-breaking-customers-functions

* Use the internal module name `sls_sdk` to access the base SDK from Lambda SDK.
* Import `serverless_aws_lambda_sdk` from the wrapper script.

### Testing done
* Unit/Integration tested.
* Dashboard integration test uses a dashboard instrumented function. The customer handler imports `serverless_sdk` and ensures its the dashboard one and not the console one.

```
➜  node git:(use-sls-sdk-internally) ✗ npx mocha test/python/aws-lambda-sdk/integration.test.js
2023-04-04T09:15:14.815Z ℹ test test uid: cihan


  Python: integration
2023-04-04T09:15:14.916Z ℹ test Creating core resources test-python-sdk-cihan
2023-04-04T09:15:50.648Z ℹ test Process function success-v3-8
2023-04-04T09:15:50.649Z ℹ test Process function success-v3-9
2023-04-04T09:15:50.649Z ℹ test Process function success-sampled
2023-04-04T09:15:50.650Z ℹ test Process function error-v3-8
2023-04-04T09:15:50.650Z ℹ test Process function error-v3-9
2023-04-04T09:15:50.650Z ℹ test Process function error_unhandled-v3-8
2023-04-04T09:15:50.650Z ℹ test Process function error_unhandled-v3-9
2023-04-04T09:15:50.651Z ℹ test Process function sdk-v3-8
2023-04-04T09:15:50.651Z ℹ test Process function sdk-v3-9
2023-04-04T09:15:50.651Z ℹ test Process function sdk-dev-mode
2023-04-04T09:15:50.651Z ℹ test Process function dashboard-v3-8
2023-04-04T09:15:50.651Z ℹ test Process function dashboard-v3-9
    ✔ success-v3-8 (22940ms)
    ✔ success-v3-9 (7660ms)
    ✔ success-sampled
    ✔ error-v3-8
    ✔ error-v3-9
    ✔ error_unhandled-v3-8
    ✔ error_unhandled-v3-9
    ✔ sdk-v3-8
    ✔ sdk-v3-9
    ✔ sdk-dev-mode
    ✔ dashboard-v3-8 (809ms)
    ✔ dashboard-v3-9
2023-04-04T09:16:22.102Z ℹ test Cleanup test-python-sdk-cihan
2023-04-04T09:16:22.418Z ℹ test Deleted 86 version of the layer test-python-sdk-cihan-internal
2023-04-04T09:16:22.445Z ℹ test Deleted 67 version of the layer test-python-sdk-cihan-external
2023-04-04T09:16:23.312Z ℹ test Detached IAM policy arn:aws:iam::692593327170:policy/test-python-sdk-cihan from test-python-sdk-cihan
2023-04-04T09:16:23.519Z ℹ test Deleted IAM role test-python-sdk-cihan
2023-04-04T09:16:24.006Z ℹ test Deleted IAM policy arn:aws:iam::692593327170:policy/test-python-sdk-cihan


  12 passing (1m)
```

### CODE REVIEW

**Comment by selcukcihan** [comment]
@medikoo I'm trying to add an integration test for the dashboard instrumented case. Do we have an example of a lambda test fixture where the lambda package contains more than one file? In this new test we'll need to ship multiple files:

Screenshot 2023-04-04 at 11 27 03

We can simplify this by mimicking the dashboard's `serverless_sdk` import, but still we'll need to have at least two files: one for the handler and another for `serverless_sdk.py` that represents the dashboard sdk.

**Comment by selcukcihan** [comment]
I'm exploring `fixturesDirname` I think this is intended for this purpose.

**Comment by selcukcihan** [comment]
Got it, the integration test bundles the whole fixtures directory 👍

**Comment by selcukcihan** [comment]
@medikoo ready for review please 🙏

File: python/packages/aws-lambda-sdk/tests/fixtures/lambdas/dashboard/handler.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import serverless_sdk
+
+
+def hello(event, context):
```
Comment (medikoo) [inline python/packages/aws-lambda-sdk/tests/fixtures/lambdas/dashboard/handler.py:5]: Can we also test access to console SDK via `sls_sdk` import?

File: python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk_sampled_out.py [hunk 2]
```diff
@@ -7,7 +7,7 @@ def handler(event, context):
     counter += 1
     invocation_id = counter
 
-    from serverless_sdk import serverlessSdk as sdk
+    from sls_sdk import serverlessSdk as sdk
```
Comment (medikoo) [inline python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk_sampled_out.py]: I think here we can rely on `serverless_sdk` (I would stick to that name wherever we test public SDK, and do not access private modules in lib)

**Review by medikoo — CHANGES_REQUESTED** [review]
Looks good 👍 I have just few suggestions

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**